src/fsharp/CompileOps.fs

```diff
@@ -3582,7 +3582,7 @@ type TcImports(tcConfigP:TcConfigProvider, initialResolutions:TcAssemblyResoluti
                   FileName=fileName;
                   ProviderGeneratedAssembly=Some theActualAssembly
                   IsProviderGenerated=true;
-                  ProviderGeneratedStaticLinkMap= if g.isInteractive then None else Some (ProvidedAssemblyStaticLinkingMap.CreateNew())
+                  ProviderGeneratedStaticLinkMap= (*if g.isInteractive then None else*) Some (ProvidedAssemblyStaticLinkingMap.CreateNew())
```

Comment:
Yep, I left it there to ensure the static link map changes were scrutinised thoroughly. I suppose the diff is good enough for that though.